core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.spec.ts

```diff
@@ -689,21 +697,4 @@ describe('Topic editor tab directive', function() {
       $scope.removeDiagnosticTestSkillDropdown();
       expect($scope.diagnosticTestSkillsDropdownIsShown).toBeFalse();
     });
-
-  it('should warning message when a skill with less than 2 questions is added' +
```

Comment:
Where's the replacement for this test?

Reply:
Now a skill with less than 2 questions is not eligible for display in the dropdown so this case is not required.
Regarding replacement can you please help me to understand which case should be considered?

Reply:
Hm, is there a test case that checks that skills with fewer than 2 questions can't be displayed in the dropdown?